### PR TITLE
[18]Software Developer Conditionals

### PR DESCRIPTION
**Pull request by Iyiola-stack**
## Task: `Created a conditional statement
- `Changes you made to given task`

### CODE REVIEW

File: submissions/peterIyiola/JS/index2.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+for (let i = 1; i <= 116; i++) 
+{
+    
+if( i % 3 ==0)   
+    console.log("software");
+
+if(i % 5 ==0)
+    console.log("Developer");
+
+if (i % 3 ==0 && i % 5 ==0)
```
Comment (adaugochi) [inline submissions/peterIyiola/JS/index2.js:10]: This should be executed first. Also, make use of `if else if and else` statement

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/peterIyiola/JS/index2.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+for (let i = 1; i <= 116; i++) 
+{
+    
+if (i % 3 ==0 && i % 5 ==0)
+    console.log("Software" + " " + "Developer");
```
Comment (adaugochi) [inline submissions/peterIyiola/JS/index2.js:5]: No need to do a concatenation here simply do `("Software Developer")`

File: submissions/peterIyiola/JS/index2.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+   
+else if( i % 3 ==0)   
+    console.log("software");
+
+else  (i % 5 ==0)
```
Comment (adaugochi) [inline submissions/peterIyiola/JS/index2.js:10]: Also, console.log(i) in your else block instead

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Iyiola-stack** [comment]
Done

File: submissions/peterIyiola/JS/index2.js (hunk 3 above)
Comment (adaugochi) [reply submissions/peterIyiola/JS/index2.js:10]: Did you forget about this condition `(i % 5 ==0)`

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Iyiola-stack** [comment]
What do you want me to do with this condition

**Comment by ToluAkin** [comment]
@Iyiola-stack Your output based on the logic of the task should be 4. Updated your logic to what @adaugochi mentioned.

**Comment by Iyiola-stack** [comment]
done

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Iyiola-stack You are still not doing what is required of the task.

**Comment by Iyiola-stack** [comment]
Please,I need clarification on what is expected of me at this point.
I'm confused really

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Iyiola-stack Please go through the comments by @adaugochi again

**Comment by ToluAkin** [comment]
@Iyiola-stack you still need the output the number if it does not match any of those criteria.

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Iyiola-stack You are meant to only out the strings when the conditions are met not the number and string

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**